Xcodes/Frontend/XcodeList/XcodeListView.swift

```diff
@@ -7,26 +7,34 @@ struct XcodeListView: View {
     @Binding var selectedXcodeID: Xcode.ID?
     private let searchText: String
     private let category: XcodeListCategory
+    private let isInstalledOnly: Bool
     
-    init(selectedXcodeID: Binding<Xcode.ID?>, searchText: String, category: XcodeListCategory) {
+    init(selectedXcodeID: Binding<Xcode.ID?>, searchText: String, category: XcodeListCategory, isInstalledOnly: Bool) {
         self._selectedXcodeID = selectedXcodeID
         self.searchText = searchText
         self.category = category
+        self.isInstalledOnly = isInstalledOnly
     }
     
     var visibleXcodes: [Xcode] {
         var xcodes: [Xcode]
         switch category {
         case .all:
             xcodes = appState.allXcodes
-        case .installed:
-            xcodes = appState.allXcodes.filter { $0.installState.installed }
+        case .release:
+            xcodes = appState.allXcodes.filter { $0.version.isNotPrerelease }
+        case .beta:
+            xcodes = appState.allXcodes.filter { $0.version.isPrerelease }
         }
         
         if !searchText.isEmpty {
             xcodes = xcodes.filter { $0.description.contains(searchText) }
         }
         
+        if isInstalledOnly {
+            xcodes = appState.allXcodes.filter { $0.installState.installed }
```

Comment:
```suggestion
            xcodes = xcodes.filter { $0.installState.installed }
```
I think it would be better if the `isInstalledOnly` is on, we filter with `AND` instead of `OR`, so that the user can use both filters.

Reply:
Good to go, this is my purpose too, I missed it.